velox/functions/prestosql/window/tests/WindowTestBase.h

```diff
@@ -48,6 +48,28 @@ static std::vector<std::string> kSortOrderBasedOverClauses = {
     "order by c1 desc nulls first, c0 asc nulls first",
 };
 
+/// Common set of window function over clauses for window frame tests to ensure
+/// total ordering for deterministic results in ROW mode.
+static std::vector<std::string> kFrameOverClauses = {
+    "partition by c0 order by c1, c2",
```

Comment:
You should probably add some that change the sort order for completeness.

Reply:
Thanks for the suggestion, added more clauses. I hadn't added more before as I was concerned the tests would time out on CircleCI.